README.md

```diff
@@ -1,2 +1,152 @@
-# school-library-management-system
-A Ruby project with classes for managing students, teachers, and their information, including personal details and permissions.
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[school-library-management-system\] ](#-school-library-management-system-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Usage](#usage)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [school-library-management-system] <a name="about-project"></a>
+
+**[school-library-management-system]** A Ruby project with classes for managing students, teachers, and their information, including personal details and permissions.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>RUBY</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">Ruby Terminal</a></li>
+  </ul>
```

Comment:
- [ ] It would be nice if this link takes me to the Ruby website and not to the PostgreSQL website 🤔 because I don't think so that Ruby has any step sis like PostgreSQL 😄 Kindly fix this link to make your readme descriptive and with the correct information 👍